source/incident-collaboration/running-playbooks.rst

```diff
@@ -100,7 +100,7 @@ If a playbook run was ended prematurely, it can be restarted within the channel
 Playbook dashboard
 ------------------
 
-Each playbook has a dashboard with details such as how many runs using the playbook are currently in progress, total runs per week over a 12-week period, and a list of runs with details such as status, duration, owner/participants, and tasks finished.
+Each playbook has a dashboard with details such as how many runs using the playbook are currently in progress, total runs per week over a 12-week period, and a list of runs with details such as status, duration, owner/participants and tasks finished.
```

Comment:
```suggestion
Each playbook has a dashboard with details such as how many runs using the playbook are currently in progress, total runs per week over a 12-week period, and a list of runs with details such as status, duration, owner/participants, and tasks finished.
```

Reply:
We use Oxford commas in our documentation - so that means a comma is placed between the second last item and the "and" preceding the last item.